source/CommonJobs/CommonJobs.Application.EvalForm/Commands/UpdateEvaluatorsCommand.cs

```diff
@@ -27,13 +27,27 @@ public override void Execute()
             {
                 if (e.Action == EvaluatorAction.Add)
                 {
-                    ExecuteCommand(new GenerateCalificationCommand(_employeeEvaluation.Period, _employeeEvaluation.UserName, e.UserName, _employeeEvaluation.TemplateId, CalificationType.Evaluator, _employeeEvaluation.Id));
+                    var employee = RavenSession
```

Comment:
Maybe this validation could be one outside `foreach` in order to access to database only one time. If it is not possible to do it right now, please add a TODO annotation.